### PR TITLE
Update Debian build to Buster and LibSSL 1.1

### LINKED ISSUE #1670: Add support for Raspbian Buster (openssl 1.1.1c)
**Describe the bug**
After installing Jellyfin on a Raspbian Buster based system, I was not able to access the webUI via browser. The service was reported as running but Jellyfin was not listening on port 8096 (other services were running just fine). Based on internet research this seems to be tied to the version of openssl that ships with Raspbian Buster (1.1.1c).

**To Reproduce**
1. Install Jellyfin as described in https://jellyfin.readthedocs.io/en/latest/administrator-docs/installing/#debian
2. Run "service jellyfin status" - the service is active
3. When you go to [Device IP]:8096, you get a generic "Unable to connect" FF error 
4. When you run "netstat -tnlp" Jellyfin is not listed, while other services (e.g. SSH server) are listed and work fine. 
5. Try to run Jellyfin manually "/usr/bin/jellyfin start"  and you'll get a "No usable version of the libssl was found" 

**Expected behavior**
Jellyfin webUI should be accessible. 

**System**
 - OS: Raspbian Buster
 - Browser: Firefox
 - Jellyfin Version: 10.3.7
 - Reverse proxy: None
 - OpenSSL 1.1.1c 28 May 2019

**Additional context**
The following article seems to describe this issue in more detail:

http://www.d0wn.com/installing-jellyfin-on-a-rasberry-pi4/ 

I did not try downgrading OpenSSL as described by the article.

### PR DESCRIPTION
**Pull request by joshuaboniface**
**Changes**
With Debian Buster being the stable release for nearly two months now, update the Debian builds to use it instead of Stretch.

For the amd64 build, move from using the official Microsoft docker image (which is just Debian anyways) to the main Debian 10 image, since Microsoft is taking their sweet time in releasing for Buster. I think this is better anyways as this way all the build images are consistent, even if it uses a bit more space.

Update the LibSSL dependencies to be only 1.1. ~I will backport the stable version from Buster into Stretch for inclusion in our repos, which should just be installed automatically on an upgrade, in order to preserve full functionality for Stretch.~ In combination with the new DotNET version, this will enable full support for LibSSL 1.1.

**Issues**
Fixes #1670

### CODE REVIEW

**Comment by joshuaboniface** [comment]
Oh, nice, libssl1.1 is just present in Stretch, so this will "just work":tm:.

**Comment by joshuaboniface** [comment]
Tested building all 3 packages; the amd64 one definitely works, but will need someone (cc @Millichrome) to test it out (`./build debian-package-armhf` or `-arm64` as required).

File: deployment/debian-package-arm64/Dockerfile.amd64 [hunk 1]
```diff
@@ -16,7 +16,7 @@ RUN apt-get update \
 
 # Install dotnet repository
 # https://dotnet.microsoft.com/download/linux-package-manager/debian9/sdk-current
-RUN wget https://download.visualstudio.microsoft.com/download/pr/69937b49-a877-4ced-81e6-286620b390ab/8ab938cf6f5e83b2221630354160ef21/dotnet-sdk-2.2.104-linux-x64.tar.gz -O dotnet-sdk.tar.gz \
+RUN wget https://download.visualstudio.microsoft.com/download/pr/228832ea-805f-45ab-8c88-fa36165701b9/16ce29a06031eeb09058dee94d6f5330/dotnet-sdk-2.2.401-linux-x64.tar.gz -O dotnet-sdk.tar.gz \
```
Comment (EraYaN) [inline deployment/debian-package-arm64/Dockerfile.amd64]: Don't they have a go.microsoft fwlink for this?

Comment (joshuaboniface) [reply deployment/debian-package-arm64/Dockerfile.amd64]: I don't know for sure, those are the direct download links.

**Comment by Millichrome** [comment]
I might be able to test the ARM build. Please let me how I can backup my current install.

Just to make sure; I would need to uninstall the current versions of Jellyfin and OpenSSL 1.x.x and then do an installation routine for a what I am assuming is a test build, right?

How do I access the test build? Is there a .deb file or how do I get it?

**Comment by joshuaboniface** [comment]
@Millichrome If you stop JF and back up `/var/lib/jellyfin` and `/etc/jellyfin` that should be sufficient to restore the stable version later. You'll be best to try tonight's nightly (install `jellyfin-nightly` from the repo, after backing up), since GitHub won't let me attach `.deb` files.

**Comment by SharerMax** [comment]
@joshuaboniface I can't find lasted nigiht version for debian armhf platform. the [repo](https://repo.jellyfin.org/releases/server/debian/versions/nightly/) is only night20190902 for debian armhf platform, and the version(night20190902) is not fix #1670 . Can you give me a link to dwonload lasted version?

**Comment by joshuaboniface** [comment]
@SharerMax Weird, it didn't build the other night but just worked for me now. It's here: https://repo.jellyfin.org/releases/server/debian/versions/nightly/20190904/

**Comment by SharerMax** [comment]
> @SharerMax Weird, it didn't build the other night but just worked for me now. It's here: [repo.jellyfin.org/releases/server/debian/versions/nightly/20190904](https://repo.jellyfin.org/releases/server/debian/versions/nightly/20190904/)

thx. It's works fine. thx again

**Pull request merged**